websockets/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -118,28 +118,38 @@ Subscriptions only exist in the context of each instance of a WebSockets Connect
 
 Triggers initiate the execution of a flow upon receiving a message or event notification. You can specify the following triggers for WebSockets-related events:
 
-* `inbound-listener` exposes an endpoint that external systems can connect to and initiates the containing flow when a message is received.
-* `on-inbound-connection` initiates a flow when a new connection to the Inbound Listener is established.
-* `outbound-listener` exposes an endpoint that initiates a flow when outbound WebSockets receive a message from an external system.
-* `on_connection-closed` initiates the containing flow when inbound or outbound WebSockets close.
+* On New Inbound Message
+  Exposes an endpoint that external systems can connect to and initiates the containing flow when a message is received.
+* On New Inbound Connection
+  Initiates a flow when a new connection to the Inbound Listener is established.
+* On New Outbound Message
+  Exposes an endpoint that initiates a flow when outbound WebSockets receive a message from an external system.
+* On Socket Closed
+  Initiates the containing flow when inbound or outbound WebSockets close.
 
 [[operations]]
 == Operations
 
 Anypoint Connector for WebSockets (WebSockets Connector) supports the following operations:
 
-* `open-outbound-socket` connects to an external system and creates an outbound webSocket.
-* `send` sends a message through a WebSocket.
-* `subscribe-groups` subscribes a WebSocket to one or more groups.
+* Open outbound socket
```

Comment:
For the other bullet points, you used title case, now here (Open outbound socket) it is sentence. If these are operation names, you can use title case for all of them.